src/components/footer/EsFooter.vue

```diff
@@ -109,12 +109,20 @@
   export default Vue.extend({
     name: 'es-footer',
 
+    props: {
+      currentRoute: {
```

Comment:
currentRoute is a living property on the $router object. Don't name it like this. Why do you need it?

Reply:
No no need anymore. Deleted.